.github/ISSUE_TEMPLATE/3_product_release_checks.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Product Release Checks
+about: Use this template to track all release checks for your component
+title: "[Product] Release Checks"
+labels: compliance, documentation, "security analysis", "test results"
+assignees: kelaja
```

Comment:
I'm not quite sure about having a specific GH user mentioned in a template. Maybe it's better that release management self-assigns it at issue creation?

Reply:
IMHO i like the idea to have dedicated people assigned while creating a issue... but get your point... 

Maybe just one responsible person is not enough.

Reply:
I would also think, that the one creating the tickets can then assign the one responsible for tracking at the moment for creation. For now, I just stayed as close to the previous version as possible and only moved stuff. But will remove it now that you also suggest that 😁

Reply:
Done in: [70def77](https://github.com/eclipse-tractusx/sig-release/pull/479/commits/70def77137e81d7d94dd1d6b60b624f0ee98c6ef)